src/main/java/org/mamute/controllers/UserProfileController.java

```diff
@@ -199,6 +200,23 @@ public void uploadAvatar(UploadedFile avatar, @Load User user) {
 		result.use(json()).withoutRoot().from(attachment).serialize();
 	}
 
+	@CustomBrutauthRules(ModeratorOnlyRule.class)
+	@Delete
+	public void delete(Long userId) {
+		if (!this.environment.supports("deletable.users")) {
+			result.notFound();
+			return;
+		}
```

Comment:
I'm not sure if we should use brutauth since this is not authorization logic. Maybe we should create an interceptor to do this. Can we leave this code improvement as a new issue?